api/osmt-dev-stack.env.example

```diff
@@ -2,7 +2,10 @@
 # Uncomment this line when you use ng serve for front end development, or export OSMT_FRONT_END_PORT=4200 to your shell
 #OSMT_FRONT_END_PORT=4200
 
-OAUTH_ISSUER=https://xxxxxx.okta.com/oauth2/default
-OAUTH_CLIENTID=xxxxxx
-OAUTH_CLIENTSECRET=xxxxxx
-OAUTH_AUDIENCE=xxxxxx
+#aa-osmt-viewer.@wgu.edu / Password123!
+#aa-osmt-admin@wgu.edu / Password123!#
+OAUTH_ISSUER=https://dev-35915850.okta.com/oauth2/default
+OAUTH_CLIENTID=0oa5ciesmoUuw6XJt5d7
+OAUTH_CLIENTSECRET=cZK22mgNATE9ylaW41kGEFYpWHajqls1VYuT3usD
+OAUTH_AUDIENCE=0oa5ciesmoUuw6XJt5d7
```

Comment:
Secrets in a commit?

Reply:
It's for a local dev test Okta server. But I'll remove it.